applications/accounts/deploy/values.yaml

```diff
@@ -34,6 +34,8 @@ harness:
       value: "user"
     - name: DB_PASSWORD
       value: "password"
+    - name: JAVA_OPTS
```

Comment:
in a more recent keycloak image version than we currently, a `JAVA_OPTS_APPEND` env variable was introduced. They introduced this because there is some risk of values being omitted if JAVA_OPTS is defined directly: https://github.com/keycloak/keycloak-containers/pull/282/files

Cannot say if this is a problem here, but might be a good idea to upgrade and use `JAVA_OPTS_APPEND`

Reply:
since we don't use the newer versions this doesn't seem a problem to me